### DISCUSSION TITLE
Captain's talent "Travelling Tradesman" doesn't give +20% item sell value

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#14312] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

it should
![Screenshot_1](https://github.com/user-attachments/assets/393f147f-f145-46a5-832e-35e42365a269)
Both of these multipliers are always zero 
https://github.com/FakeFishGames/Barotrauma/blob/0aee53976202c7ea5c0bb3f26669a97c8dc2400c/Barotrauma/BarotraumaShared/SharedSource/Map/Map/Location.cs#L339-L344
It can be "fixed" by 
![Screenshot_2](https://github.com/user-attachments/assets/e5ba68df-d9f1-417b-8a36-dd761488dcf9)
Although I don't know how it works, why it helps or why PermanentStat doesn't work
![barodev small](https://github.com/user-attachments/assets/136201e5-b967-4294-981f-f0391a91dc72)

### Reproduction steps

Start new campaign
giveexperience 1000000
unlock this talent on captain
Your revolver rounds now should cost 6mk instead of 5mk

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix.
Value addition is now applied properly and taken away if the character dies.
Closing discussion as resolved
